components/public-api/typescript-common/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@gitpod/public-api-common",
```

Comment:
@akosyakov Personally I would love to see a README.md next to this file, which explains the "WHY does this package exist, how it compares to gitpod-protocol, and "which package to choose for what"".

Can be a follow-up :+1: